apps/web/src/pages/PoolCandidates/ViewPoolCandidatePage/components/CareerTimelineSection/CareerTimelineSection.tsx

```diff
@@ -33,6 +36,15 @@ const CareerTimelineSection = ({ experiences }: CareerTimelineSectionProps) => {
 
   return (
     <>
+      {features.recordOfDecision && (
+        <Heading Icon={UserCircleIcon} color="tertiary">
+          {intl.formatMessage({
+            defaultMessage: "Career timeline",
+            id: "5FK/4R",
+            description: "Heading for the users experiences",
```

Comment:
Was this description copy and paste from somewhere else?

Reply:
maybe "Heading for the career timeline section"? anyway, in the long run, this should be consolidated with all of the other "Career timeline" strings. nevermind!